annotations/update_pending_annotations.py

```diff
@@ -0,0 +1,158 @@
+""" Update Restricted Publication Annotations 
+
+This script will update preivously 'Restricted' Publications
+with their updated annotations after they have become 'Open Access' """
+
+import synapseclient
+from synapseclient import Table, RowSet
+import argparse
+import pandas as pd
+from attribute_dictionary import PUBLICATION_DICT
+""" Login to Synapse """
```

Comment:
Whoops, I think this docstring should be moved _into_ the function 😅

Reply:
Yep, you are right.